src/assets/i18n/fr.json

```diff
@@ -223,14 +152,47 @@
   "confirm-mail-text": "Un code d’activation a été envoyé à l’adresse : monadresse@mail.com. Vous pourrez vous connecter dès que vous aurez validé le lien d’inscription. ",
   "auth_code": " Code d'authentification",
   "send_code": "Renvoyer le code",
+  "EMAIL": "VEUILLEZ ENTRER VOTRE COURRIEL",
+  "PASSWORD":"MOT DE PASSE",
   "confirm_email": "Confirmer mon mail",
   "confirm-mail-text2": "Ce code ne sera valide que pendant 5 min. Vous pouvez demander à renvoyer un nouveau code en cliquant sur sur le bouton ci-dessous :",
-  "afficherWallet": "AFFICHER LE PORTEFEUILLE",
-  "GAS_TAKEN_FOR_THE_TRANSACTION": "Gaz prélevé pour la transaction",
-  "Ajout de fichiers": "Ajout de fichiers",
-  "Ajouter des liens": "Ajouter des liens",
-  "Ajouter un lien": "Ajouter un lien",
   "All_campaign": "Toutes les campagnes",
+  "circulating_supply":"ALIMENTATION CIRCULANTE",
+  "Fully_diluted_market_cap":"CAPITALISATION BOURSIÈRE APRÈS DILUTION",
+  "MARKET_CAP":"CAP MARCHE",
+  "FARM_YOUR_SOCIAL_NETWORKS":"EXPLOITER VOS RÉSEAUX SOCIAUX",
+  "earn_crypto":"Gagner des cryptos avec un post",
+  "Launch_App_Welcome":"lancer App",
+  "New_AdPools_welcome":"nouveaux AdPools",
+  "All_Adpools_Welcome":"Tous les Adpools",  
+  "Launch_your_adpool":"Lancez votre AdPool maintenant !",
+  "Apply_to_launch":"Postuler pour lancer",
+  "Wallet_updated_FOOTER1":"Adpool SaTT est un outil innovant qui vous permet de maximiser l'impact de vos campagnes marketing. ",
+  "Wallet_updated_FOOTER2":" Il vous permet de faire de la publicité pour des influenceurs afin qu'ils parlent",
+  "Wallet_updated_FOOTER3":"de votre marque ou de votre produit, en utilisant la technologie blockchain.",
+  "Wallet_updated_FOOTER4":"En demandant la création d'un Adpool, notre équipe expérimentée vous contactera ",
+  "Wallet_updated_FOOTER5":"et vous aidera à mettre en place votre campagne. ",
+  "Wallet_updated_FOOTER6":"Faites confiance à SaTT pour une campagne marketing réussie !",
+  "Reach":"Atteinte",
```

Comment:
"Reach"  : "Reach",

Reply:
Please ensure that variable names follow the camelCase convention. This is a common standard in JavaScript where the first letter of the word is in lowercase, and each subsequent word starts with an uppercase letter. For example, use transactionAmount, transactionDate, transactionDetails, etc. This helps to maintain consistency and readability throughout the codebase."